change/@microsoft-teams-js-9c1b77aa-a0e6-4099-9eae-9fff30c8f94e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Dialog.open function takes an extra optionl parameter named messageFromChildHandler and returns a handler back instead of a ChildAppWindow.",
```

Comment:
"optional" is missing an a

Reply:
Thank you for catching

Reply:
My excuse is English is not my first language ;)